Assets/Scripts/UI/ObjectiveList/ObjectiveList.cs

```diff
@@ -38,11 +49,17 @@ public void DisplayObjectiveText(string textToSet)
 
     public void CrossOutObjectiveText()
     {
+        if (!root.GetBool("slideOut"))

```

Comment:
If it was hidden beforehand, should we re-hide it for them after a couple seconds? Maybe 5 seconds or so?